test/verify_vertices.jl

```diff
@@ -5,8 +5,8 @@ using ImmutableArrays
 # Shift the axes so that they do not line up with
 # integer values
 Δ = 0.01
-X = [0:Δ:4] + π
-Y = [0:Δ:3] + φ
+X = [0:Δ:4;] + π
```

Comment:
I know that this is the syntax that is recommended by the deprecation message, but I personally just never liked it, and much prefer to create arrays from ranges using `collect` instead; that is, this line becomes `X = collect(0:Δ:4) + π`. If you think this is too nitpicky, tell me and I'll fix it myself :)